lib/OpenQA/Schema/Result/JobModules.pm

```diff
@@ -119,8 +119,13 @@ sub details() {
         next unless $img->{screenshot};
         my $link = readlink($dir . "/" . $img->{screenshot});
         next unless $link;
-        $img->{md5_basename} = basename($link);
-        $img->{md5_dirname}  = basename(dirname($link));
+        my $base = basename($link);
+        my $dir  = dirname($link);
+        $dir =~ s,^.*/images/,,;
+        if ($dir ne 'ad') {    # avoid problem with ad block
+            $img->{md5_dirname}  = $dir;
+            $img->{md5_basename} = $base;
+        }
```

Comment:
this is like a very specific super special workaround which will bite us in the ass eventually, I assume. I'd rather ask every user to disable adblock on openqa hosts until we have a proper avoidance using migration of current results.